src/components/Facility/ConsultationCard.tsx

```diff
@@ -53,16 +54,39 @@ export const ConsultationCard = (props: ConsultationProps) => {
         )}
       </DialogModal>
       <div className="pb-16 block relative cursor-pointer border-l-2 px-4 border-l-secondary-300 hover:border-primary-500 transition-all before:absolute before:-left-[7px] before:top-0 before:w-3 before:aspect-square before:bg-secondary-400 before:rounded-full hover:before:bg-primary-500 before:transition-all">
-        <Chip
-          size="small"
-          variant={itemData.suggestion === "A" ? "alert" : "primary"}
-          text={
-            itemData.suggestion === "A" ? t("ip_encounter") : t("op_encounter")
-          }
-          className="-translate-y-2"
-        />
-        <div className="text-sm">
-          {dayjs(itemData.created_date).format("DD/MM/YYYY")}
+        <div className="flex flex-wrap items-center gap-4 mt-4">
+          <div className="flex flex-col gap-1">
+            <Chip
+              size="small"
+              variant={itemData.suggestion === "A" ? "alert" : "primary"}
+              text={
+                itemData.suggestion === "A"
+                  ? t("ip_encounter")
+                  : t("op_encounter")
+              }
+              className="-translate-y-2"
+            />
+            <div className="text-sm">
+              {dayjs(itemData.created_date).format("DD/MM/YYYY")}
+            </div>
+          </div>
+
+          {/* Patient Status Section */}
+          <div className="flex items-center gap-2 -translate-y-5 ">
+            <div className="text-sm font-normal leading-5">
+              {t("patient_status")}
+            </div>
+            <Chip
+              size="medium"
+              variant="custom"
+              className={
+                !isLastConsultation
+                  ? "bg-blue-100 text-blue-800"
+                  : "bg-red-100 text-red-800"
+              }
+              text={!isLastConsultation ? "LIVE" : "DISCHARGED"}
+            />
```

Comment:
this just checks for whther the encounter is last consultation or not right? and doesn't check if the last consultation is discharged or not.

This is not the right logic.

<img width="1790" alt="image" src="https://github.com/user-attachments/assets/159a4f50-1e91-4310-b9d5-2739fcbfa2a1" />